sqlglot/expressions.py

```diff
@@ -6871,7 +6871,14 @@ def column(
     return this
 
 
-def cast(expression: ExpOrStr, to: DATA_TYPE, copy: bool = True, **opts) -> Cast:
+def cast(
+    expression: ExpOrStr,
+    to: DATA_TYPE,
+    force_cast: bool = False,
+    overwrite_cast: bool = False,
```

Comment:
where this ever used? call it overwrite

Reply:
This isn't used anywhere, but I thought you meant to add this flag in (1), maybe I misinterpreted your suggestion?

> and then allow for flags to 1. change the type, 2. allow users to add cast no matter what

Seemed like a reasonable setting but I'm happy to remove.

Reply:
no because cast_unless had the api 

cast_unless(x, to:int, unless: varchar)

you don't have the unless argument that we used to have

Reply:
so it should be unless: t.Optional[DATA_TYPE]

Reply:
The `unless` part wasn't used anywhere in the codebase as well, as in, in all cases `unless` matched `to`. Do you think we should still have it?

Reply:
i guess we can delete both

Reply:
got rid of them both